blogposts/2022/introducting-migrator-service.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'Introducing the migrator service'
+author: Eric Fritz
+authorUrl: https://eric-fritz.com
+publishDate: 2022-02-15T12:00-00:00
+tags: [blog]
+slug: introducing-migrator-service
+heroImage: https://sourcegraphstatic.com/blog/introducing-migrator-service/introducing-migrator-service.jpg
+socialImage: https://sourcegraphstatic.com/blog/introducing-migrator-service/introducing-migrator-service.jpg
```

Comment:
- [ ] Need her+social images.

Reply:
@efritz I have a request out for images but we'll have to add them after the fact. It will be at least a few days, if not a week, unless maybe you have some temporary idea? Like a screenshot or diagram or something?

Reply:
I don't have anything prepared here, sorry 😿

Reply:
No worries, didn't expect you to! I sent over some concepts to our graphic team so we'll be able to add something in post in like a week.

Reply:
Do we need to put a placeholder here? I'm not sure it can be blank.

Reply:
Oh, I'm not actually sure! Looking back it does look like even old blog posts pre-graphics help had something

Reply:
Not exciting but we could just do the logo https://about.sourcegraph.com/sourcegraph-mark.png